test/test_retriever.py

```diff
@@ -691,3 +692,19 @@ def test_setup_functions():
     assert os.path.exists(raw_dir_files.format(file_name="")) is False
     setup_functions()
     assert os.path.exists(raw_dir_files.format(file_name=""))
+
+
+def test_reset_retriever(tmpdir):
```

Comment:
Let us move it above the function `def test_setup_functions():`
here https://github.com/weecology/retriever/pull/1304/files#diff-889328868e03c9f5ac1fcaafa693558dL685

Reply:
Done.